krabs/krabs/etw.hpp

```diff
@@ -77,6 +77,16 @@ namespace krabs { namespace details {
          */
         EVENT_TRACE_PROPERTIES query();
 
+        /**
+         * <summary>
+         * Configures the ETW trace session settings.
+         * </summary>
```

Comment:
let's update the comment with a link to https://docs.microsoft.com/en-us/windows/win32/api/evntrace/nf-evntrace-tracesetinformation so that users know how to configure these parameters

Reply:
Done.